javaHapiValidatorLambda/pom.xml

```diff
@@ -0,0 +1,185 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Did we check the licenses of these new dependencies?

Reply:
Yes, they are ok.

They look like many packages, but most come from either aws-lambda-java-libs or HAPI that are both Apache 2.0